neurons/text/prompting/validators/core/neuron.py

```diff
@@ -348,7 +348,8 @@ def inference(
             roles = roles, 
             messages = contents, 
             uids = uids, 
-            timeout = timeout
+            timeout = timeout,
+            priority = 2 # the priority of inference is always higher then training

```

Comment:
Should inference take priority over training? Wouldnt this create issues where the model is always in inference?